schematics/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "schematics" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a40b20635c0e43d18d3aff76220f6cd95ea4decb3f37765e49529b17d81b0439
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - schematics
+
+about:
+  home: https://github.com/schematics/schematics
+  license: BSD
```

Comment:
This one has to be out of https://spdx.org/licenses/, if they don't specify it you may have to dig a little. This field is super important to the conda-forge reviewers